cmd/build/buildv2.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/okteto/okteto/pkg/cmd/build"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/registry"
+	"github.com/okteto/okteto/pkg/types"
+)
+
+// BuildV2 builds the images defined by a manifest
+func (bc *Command) BuildV2(ctx context.Context, manifest *model.Manifest, options *types.BuildOptions) error {
+	toBuildSvcs := getToBuildSvcs(manifest, options)
```

Comment:
what about the services passed as args ? a user can select only 1 or multiple services from the manifest, not necesary are all of them

Reply:
I added them on the options. Function `getToBuildSvcs` checks if options args is not empty and return it, so that case should be covered

Reply:
i did not saw that! awesome!